agenta-web/src/components/Evaluations/Evaluations.tsx

```diff
@@ -65,6 +65,8 @@ export default function Evaluations() {
     const [sliderValue, setSliderValue] = useState(0.3)
 
     const [evaluationError, setEvaluationError] = useState("")
+    const [btnText, setBtnText] = useState("")
```

Comment:
Instead of maintaining three separate states, would be better to maintain a state object to make things concise and reduce rerenders:

`const [error, setError] = useState({
    message: "",
    btnText: "",
    endpoint: ""
})`